src/main/webapp/app/exam/participate/exam-participation.service.ts

```diff
@@ -9,11 +9,11 @@ import { catchError, map } from 'rxjs/operators';
 import { ExerciseService } from 'app/exercises/shared/exercise/exercise.service';
 import { Exam } from 'app/entities/exam.model';
 import * as moment from 'moment';
-import { getLatestSubmissionResult } from 'app/entities/submission.model';
+import { getLatestSubmissionResult, Submission } from 'app/entities/submission.model';
```

Comment:
`'Submission' is defined but never used`

Reply:
done